trace_context/README.md

```diff
@@ -0,0 +1,28 @@
+# Trace Context
+
+The trace context positions an incoming request in potentially multiple trace
+graphs. For example, an incoming request might have information about its
+gateway in one service, and information from its direct upstream in another.
+
+Two propagation fields carry the common and vendor-specific properties that
+make up the trace context.
+
+* `Trace-Parent` describes the position of the incoming request in its trace graph in a portable, fixed-length format. Its design focuses on fast parsing.
+
+* `Trace-State` maps all graphs the incoming parent is a part of in potentially vendor-specific formats. For example, if an ancestor of this request was in a different trace, there will be a separate entry for the last position it was in that trace graph.
```

Comment:
I'm having trouble parsing the first sentence here. I think it could benefit from a rewording. I'd suggest something but I'm not sure exactly what it's trying to say.

Reply:
reworded